src/MpesaDaraja/Http/MpesaOnline/STKPush.php

```diff
@@ -14,7 +14,22 @@
 class STKPush extends AbstractDarajaQuery
 {
     protected string $endpoint = 'mpesa/stkpush/v1/processrequest';
-    
+
+    protected array $validationRules = [
+        'TransactionType' => 'required|string|exists_in:TransactionType_stkpush',
+        'Amount' => 'required|numeric|lte:max_txn|gte:min_txn',
+        'PartyA' => 'required|numeric|phone',
+        'PhoneNumber' => 'required|numeric|phone',
```

Comment:
You can add a validation to check phone number to have atleast 12digits

  => 'required|numeric|min:12'

Reply:
ooh yeah this is taken care of via the phone rule